app/src/app/[lng]/auth/login/page.tsx

```diff
@@ -65,6 +65,22 @@ export default function Login({
     }
   }
 
+  // Extract doesInvitedUserExist from callback params\
+  // If it is true, redirect to /user/invites page
+  // If it is false, redirect to /auth/signup page
+  const [path, queryString] = callbackUrl.split("?");
```

Comment:
I think we need to check here if there is a `?` in the string beforehand? Since people can navigate directly to the login page (we had errors about that previously).